lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -360,24 +360,20 @@ public function getCreatePrimaryKeySQL(Index $index, $table)
      */
     protected function getCreateColumnCommentSQL($tableName, $columnName, $comment)
```

Comment:
I belive, there is the same issue with `getAlterColumnCommentSQL`, `getDropColumnCommentSQL`, `getCommentOnTableSQL`

-> we should update out functional tests to test at least create/alter/drop comment with quote table name for all platforms